droid-configs.inc

```diff
@@ -514,7 +514,7 @@ fi
 %post
 # Force an update of ssu repositories
 touch %{board_mapping_dir}/*
-touch %{_datadir}/ssu/features.d/*
+touch %{_datadir}/ssu/features.d/* || :
```

Comment:
Also doesn't this need &> /dev/null || : ? The output will come there even if there is || : right?